### DISCUSSION TITLE
Starting page is practically unusable

### DISCUSSION THREAD

**Discussion by SFort** [community/community#15441] [Other Feature Feedback, Questions, & Ideas]
The starting page has been compleatly replaced by tutorial popups.
The new beta feed shows nothing of interest/relavance and i'm fairly certain most users know what they want in their feed already.

![screenshot](https://user-images.githubusercontent.com/1879846/164882792-03ddb869-9d17-49a9-80d6-f5587a5d09f3.png)

**Comment by timlg07** [comment]
You can just remove them from the dashboard and end up with the usual landing page. And I think they're useful to have for new users.

**Reply by SFort** [reply]
i have no idea how to turn these off, maybe they should add another tutorial.

it feels like every other time i open github there's an amazing new feature used by a fraction of the users that needs to be turned off / patched out with a plugin, with no way to blanket disable them.
even making new repos is now a needless inconveniance, it takes me 5min each time to disable all the clutter i don't want. it's bad design.
(not to mention some of them used to redirect you to the homepage, and a few things that shouldnt still do)

the point isn't that the feature/s is useless for everyone but that it's useless for the majority and in this perticular case they could have easily excluded older accounts / placed the help in a more senseble location

**Reply by timlg07** [reply]
Turning these off is very straightforward IMO. Just click on the three dots in the upper right corner of each section and hit "Remove from Dashboard" (or similar, can't remember what it said exactly).   
It's a bit hidden, but not necessarily bad design.

I totally agree on the last part though. It should've been disabled by default for older accounts and maybe just show us a tiny option to enable/ navigate to the help section if required.

**Reply by SFort** [reply]
thanks! that explains that
(i tend to disable svgs since it doesn't break functionallity and removes a good amount of ui clutter, in this case it actually broke functionality)

**Reply by timlg07** [reply]
ah, I see \^\^  
(and it looks a bit like you're using the web in a terminal haha)

**Reply by flatrocks** [reply]
I guess we should at least be thankful it's not Clippy.  "_It looks like you're trying to create a new repository.  Do you want me to help?_"

**Reply by timlg07** [reply]
true, I hate things like popups saying this. Clippy was cute tho, I miss him

**Comment by jpluscplusm** [comment]
They've clearly been put there in error. The 3-dots menu tells me why they're there: apparently I've not opened a PR or created a repo recently.

Except I'm pretty sure I've done both of those, pretty recently!

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
